packages/discordx/examples/slash/commands/group.ts

```diff
@@ -19,7 +19,7 @@ enum TextChoices {
 @SlashGroup({ name: "text", root: "testing" })
 export abstract class Group {
   @Slash("voicechannel")
-  @SlashGroup({ name: "maths", root: "testing" })
+  @SlashGroup("maths", "testing")
```

Comment:
Is this because you can have a subgroup and a group with the same name?

Reply:
you can have them with same name, it's just point to Level 1 or Level 2 position in group.

Reply:
Yeah, I just wanted to make sure that it was actually necessary to specify root again when assigning. This is just one more point towards having group definitions separately.

Reply:
Yeah, see docs update. Developers will understand, what is level 1 and level 2, when they read from start.

![image](https://user-images.githubusercontent.com/43092101/154216908-1c419044-84b9-444a-8740-1090ec44d8c5.png)